.github/workflows/build-and-publish.yml

```diff
@@ -7,49 +7,88 @@ on: [workflow_dispatch]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  common:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get Changelog Entry
+      id: changelog_reader
+      uses: mindsers/changelog-reader-action@v2
+      with:
+        validation_depth: 10
+        path: ./CHANGELOG.md
+    - name: Get Version Info
+      id: read_metadata
+      run: |
+        echo "Version: ${{ steps.changelog_reader.outputs.version }}"
+        echo "Changes: ${{ steps.changelog_reader.outputs.changes }}"
+        $owner = "${{ github.repository_owner }}".ToLower()
```

Comment:
💡observation/question - In bash Assignment of variables with space after the (=) sign, noticed spaces here  `$owner = "${{ github.repository_owner }}".ToLower()` - Is it because now we use `"PowerShell --> under ubuntu-latest"`?

Reply:
Yeah, exactly. These workflows have jobs that run on different platforms, and using PowerShell is the simplest way to make the scripting syntax the same for all of them. For consistency, I thought it'd be best for _all_ the jobs to use PowerShell, even if they didn't need to.

Also, although PowerShell doesn't care whether there's a space or not, I quite like having it there because it makes it (slightly) more obvious it's not bash, and might save us in future from adding bash syntax and wondering why it isn't working...